crt_portal/cts_forms/management/commands/flag_repeat_writers.py

```diff
@@ -0,0 +1,53 @@
+from django.db import connection
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Flag reports that were submitted by repeat writers."
+
+    def handle(self, *args, **options):
+        repeat_summary_sql = """
+        WITH repeat_summary AS (SELECT violation_summary, COUNT(*)
+        FROM cts_forms_report
+        GROUP BY violation_summary
+        HAVING count(*) > 50 )
```

Comment:
Do we want this to be greater than or equal to 50? Right now won't it only pull in reports with a count of 51 and higher?

Reply:
Good point.  I will change that.